### PR TITLE
don't output message when not installing kiali

### LINKED ISSUE #1331: Confusing output from Kiali operator install "Will use the last Kiali release:..."
During the install of the Kiali operator, I'm seeing a line printed out that may be confusing for end users.  I installed Kiali operator v1.0.0 as part of service mesh tp12.

I use this command:
`bash <(curl -L https://git.io/getLatestKialiOperator) --operator-image-version v1.0.0 --operator-watch-namespace '**' --accessible-namespaces '**' --operator-install-kiali false`

The operator is installed successfully, but I notice this line in the output:
`Will use the last Kiali release: v1.3.0`

I verified that the v1.0.0 version of the operator was actually installed.  However, this output line may cause confusion.  Best I can tell is that the environment variable KIALI_IMAGE_VERSION is not being set and this causes the line to be printed in the output.

https://github.com/kiali/kiali/blob/master/operator/deploy/deploy-kiali-operator.sh#L646-#L655

My output for reference:
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 45872  100 45872    0     0   100k      0 --:--:-- --:--:-- --:--:--  100k
Using 'oc' located here: /usr/bin/oc
=== UNINSTALL SETTINGS ===
UNINSTALL_EXISTING_KIALI=
UNINSTALL_EXISTING_OPERATOR=
UNINSTALL_MODE=
=== UNINSTALL SETTINGS ===
envsubst is here: /usr/bin/envsubst
Using downloader: wget -q -O -
Will use the last Kiali release: v1.3.0
IMPORTANT! The Kiali operator will be given permission to create cluster roles and
cluster role bindings in order to grant Kiali access to all namespaces in the cluster.
=== OPERATOR SETTINGS ===
OPERATOR_IMAGE_NAME=quay.io/kiali/kiali-operator
OPERATOR_IMAGE_PULL_POLICY=IfNotPresent
OPERATOR_IMAGE_VERSION=v1.0.0
OPERATOR_INSTALL_KIALI=false
OPERATOR_NAMESPACE=kiali-operator
OPERATOR_SKIP_WAIT=false
OPERATOR_VERSION_LABEL=v1.0.0
OPERATOR_WATCH_NAMESPACE=""
OPERATOR_ROLE_CLUSTERROLES=- clusterroles
OPERATOR_ROLE_CLUSTERROLEBINDINGS=- clusterrolebindings
=== OPERATOR SETTINGS ===
Deploying Kiali operator to namespace [kiali-operator]
Applying yaml from URL via: [wget -q -O - https://raw.githubusercontent.com/kiali/kiali/v1.0.0/operator/deploy/namespace.yaml] to namespace [kiali-operator]
namespace/kiali-operator created
Applying yaml from URL via: [wget -q -O - https://raw.githubusercontent.com/kiali/kiali/v1.0.0/operator/deploy/crd.yaml] to namespace [kiali-operator]
customresourcedefinition.apiextensions.k8s.io/monitoringdashboards.monitoring.kiali.io created
customresourcedefinition.apiextensions.k8s.io/kialis.kiali.io created
Applying yaml from URL via: [wget -q -O - https://raw.githubusercontent.com/kiali/kiali/v1.0.0/operator/deploy/service_account.yaml] to namespace [kiali-operator]
serviceaccount/kiali-operator created
Applying yaml from URL via: [wget -q -O - https://raw.githubusercontent.com/kiali/kiali/v1.0.0/operator/deploy/role.yaml] to namespace [kiali-operator]
clusterrole.rbac.authorization.k8s.io/kiali-operator created
Applying yaml from URL via: [wget -q -O - https://raw.githubusercontent.com/kiali/kiali/v1.0.0/operator/deploy/role_binding.yaml] to namespace [kiali-operator]
clusterrolebinding.rbac.authorization.k8s.io/kiali-operator created
Applying yaml from URL via: [wget -q -O - https://raw.githubusercontent.com/kiali/kiali/v1.0.0/operator/deploy/operator.yaml] to namespace [kiali-operator]
deployment.apps/kiali-operator created
Waiting for the operator to start..
The Kiali operator is installed!

### PR DESCRIPTION
**Pull request by jmazzitelli**
fixes #1331

### CODE REVIEW

**Comment by jmazzitelli** [comment]
When Kiali is to be installed:

```
$ ./deploy-kiali-operator.sh 
Using 'oc' located here: /home/jmazzite/bin/oc
=== UNINSTALL SETTINGS ===
UNINSTALL_EXISTING_KIALI=
UNINSTALL_EXISTING_OPERATOR=
UNINSTALL_MODE=
=== UNINSTALL SETTINGS ===
envsubst is here: /usr/bin/envsubst
Using downloader: wget -q -O -
Will use the last Kiali operator release: v1.4.0
Will use the last Kiali release: v1.4.0
...
```

When Kiali is not to be installed:

```
$ ./deploy-kiali-operator.sh -oik false
Using 'oc' located here: /home/jmazzite/bin/oc
=== UNINSTALL SETTINGS ===
UNINSTALL_EXISTING_KIALI=
UNINSTALL_EXISTING_OPERATOR=
UNINSTALL_MODE=
=== UNINSTALL SETTINGS ===
envsubst is here: /usr/bin/envsubst
Using downloader: wget -q -O -
Will use the last Kiali operator release: v1.4.0
...
```

File: operator/deploy/deploy-kiali-operator.sh [hunk 1]
```diff
@@ -643,14 +643,16 @@ fi
 
 # If asking for the last release of Kiali (which is the default), then pick up the latest release.
 # Note that you could ask for "latest" - that would pick up the current image built from master.
```
Comment (jshaughn) [inline operator/deploy/deploy-kiali-operator.sh]: This is confusing.  I think we should deprecate the "latest" and "lastrelease" versions in favor of "master" and "release".  And keep "dev".

Comment (jmazzitelli) [reply operator/deploy/deploy-kiali-operator.sh]: "latest" is the common industry term that everyone knows about (think docker and the semantics about the "latest" image tag - that is the "latest" this is talking about).

"lastrelease" is simply what it refers to - the "last release" of the Kiali image.

Comment (jshaughn) [reply operator/deploy/deploy-kiali-operator.sh]: It's not the last release.  It's the most recent release.  Last means last, as in no more to follow.  Anyway, the industry is confused ;)

**Pull request merged**